Cargo.toml

```diff
@@ -27,6 +27,6 @@ tracing = "0.1"
 vbs = "0.1.4"
 
 [dev-dependencies]
-hotshot-example-types = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.48" }
```

Comment:
Also update git URL?

Reply:
Yea its just copy pasta, they resolve to the same place. But consistency is good. fixed: fba1836bd3fc8c01f34108ae2fa2bf2708688063